documentation/src/main/resources/theme/fonts/LICENSE-Overpass.txt

```diff
@@ -0,0 +1,92 @@
+Copyright (c) 2016 by Red Hat, Inc. All rights reserved.
```

Comment:
This worries me, is this license compatible with Apache license 2? We we really need to use this font?

Reply:
https://github.com/datastax/cassandra-quarkus/pull/9#discussion_r423068194